service_event/src/main/scala/no/uio/musit/microservice/event/service/EventOrderingUtils.scala

```diff
@@ -0,0 +1,64 @@
+package no.uio.musit.microservice.event.service
+
+import java.sql.{ Date, Timestamp }
+
+import no.uio.musit.microservice.event.domain.Event
+
+/**
+ * Created by jarle on 23.08.16.
+ */
+
+object EventOrderingUtils {
+
+  implicit def optionOrdering[A](implicit ao: Ordering[A]): Ordering[Option[A]] =
+    new Ordering[Option[A]] {
```

Comment:
I think this code can be made a little bit cleaner (and easier to read) by not using pattern matching on `optY`
```scala
implicit def optionOrdering[A](implicit ao: Ordering[A]): Ordering[Option[A]] =
  new Ordering[Option[A]] {
    override def compare(optX: Option[A], optY: Option[A]): Int = {
      optX match {
        case Some(x) =>
          optY.map(y =>  ao.compare(x, y)).getOrElse(1)
        case None =>
          optY.map(_ => -1).getOrElse(0)
      }
    }
  }
```